src/OrchardCore.Modules/OrchardCore.Users/Recipes/CustomUserSettingsStep.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Newtonsoft.Json.Linq;
+using OrchardCore.ContentManagement;
+using OrchardCore.Recipes.Models;
+using OrchardCore.Recipes.Services;
+using OrchardCore.Users.Models;
+using YesSql;
+
+namespace OrchardCore.Users.Recipes
+{
+    /// <summary>
+    /// This recipe step updates the custom user settings.
+    /// </summary>
+    public class CustomUserSettingsStep : IRecipeStepHandler
+    {
+        private ISession _session;
+
+        public CustomUserSettingsStep(ISession session)
+        {
+            _session = session;
+        }
+
+        public async Task ExecuteAsync(RecipeExecutionContext context)
+        {
+            if (!string.Equals(context.Name, "custom-user-settings", StringComparison.OrdinalIgnoreCase))
```

Comment:
`String`.
Where is `custom-user-settings` coming from? do you need `StringComparison.OrdinalIgnoreCase?

Reply:
Following a conventions. I haven't really put any thought into this.